.github/workflows/post-release.yml

```diff
@@ -33,12 +37,37 @@ jobs:
         uses: docker/login-action@v1
         with:
           registry: ghcr.io
-          username: ${{ secrets.CR_USER }}
```

Comment:
why was this removed?

Reply:
confusion about possibly not needing creds for public repos/artifacts... replacing!